_posts/2017/2017-02-20-continuous-documentation-with-ansible.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: post
+title: "Continuous Documentation with Ansible"
```

Comment:
I would very much prefer to have all the links and keywords and everything on one physical web-page, so that it has at least somewhat of a chance to be relevant for search engines.
Besides that I think the length (2200 words) [is just fine](https://www.snapagency.com/blog/blog-posts-2017-whats-best-length-seo)